packages/admin-ui/src/pages/packages/components/Info/ContainerList.tsx

```diff
@@ -65,7 +66,22 @@ export const ContainerList = ({ dnp }: { dnp: InstalledPackageData }) => {
           </span>
 
           {allContainersRunning ? (
-            <MdPauseCircleOutline onClick={() => onStartStop()} />
+            <MdPauseCircleOutline
+              onClick={() =>
```

Comment:
I would move as much logic as possible outside of the HTML markup to make thing easier to understand